### PR TITLE
Add support for GHC 7.10

### PR DESCRIPTION
**Pull request by wereHamster**

### CODE REVIEW

File: test/Data/Time/Format/HumanSpec.hs [hunk 1]
```diff
@@ -89,4 +96,4 @@ spec = describe "humanReadableTime'" $ do
         humanReadableTime' t n `shouldBe` "on Dec 30, 2025"
 
 parseTime' :: String -> UTCTime
-parseTime' = fromJust . parseTime defaultTimeLocale "%F %T%Q"
+parseTime' = fromJust . parseTimeM True defaultTimeLocale "%F %T%Q"
```
Comment (pbrisbin) [inline test/Data/Time/Format/HumanSpec.hs]: What does this change do?

Comment (wereHamster) [inline test/Data/Time/Format/HumanSpec.hs]: No functional change. Just that `parseTime` was deprecated in time-1.5 in favour of `parseTimeM`.

Comment (pbrisbin) [inline test/Data/Time/Format/HumanSpec.hs]: Ah gotcha

**Comment by pbrisbin** [comment]
This looks good. There's a lot of machinery in the travis file that I might not otherwise care to have, but since you've done the work, I'll take it :)

I'll merge on Green

**Comment by wereHamster** [comment]
I've based the travis config file on https://github.com/hvr/multi-ghc-travis, which adds support for more GHC versions (and also GHC head) and does a few more checks than the standard travis-ci haskell config (eg. runs `cabal check`, and attempts to build/install the dist tarball).

**Comment by pbrisbin** [comment]
By the way, kudos for a clean PR and nice commits.

**Pull request merged**